inst/tests/tests.Rraw

```diff
@@ -9691,7 +9649,7 @@ nqjoin_test <- function(x, y, k=1L, test_no, mult="all") {
   gc()  # no longer needed but left in place just in case, no harm
 }
 
-dt1 = nq_fun(400L)
+dt1 = nq_fun(100L)  # 400 reduced to 100, #5517 
```

Comment:
Few lines below we have set.seed which, I believe, together with this particular size of data was testing some particular code. Changing the size is likely to disable tests that were designed for this particular size.
I recall non equi join testing was a long process, and I was running many random data, compare against sqlite, and report back any issues to Arun. So changing the size is likely to ignore those precisely reported cases.

Reply:
added note to benchmark.Rraw